src/test/functional/functional-tests.js

```diff
@@ -751,6 +751,31 @@ describe('functional tests', function() {
       })
     })
 
+    step(`presignedUrl(httpMethod, bucketName, objectName, expires, cb)_httpMethod:GET, bucketName:${bucketName}, objectName:${_1byteObjectName}, expires:86400, requestDate:StartOfDay_`, done => {
```

Comment:
presignedUrl() is an undocumented internal function. Hence this functional test need can be deleted.

Reply:
Hi @krishnasrinivas,

This function is public documented here: https://docs.minio.io/docs/javascript-client-api-reference.html#presignedUrl